cli/cmd/agent_aws-install_ec2ssm.go

```diff
@@ -0,0 +1,347 @@
+//
+// Author:: Nicholas Schmeller (<nick.schmeller@lacework.net>)
+// Copyright:: Copyright 2022, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"fmt"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/service/iam/types"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	ssmtypes "github.com/aws/aws-sdk-go-v2/service/ssm/types"
+	"github.com/gammazero/workerpool"
+	"github.com/lacework/go-sdk/lwrunner"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	agentInstallAWSSSMCmd = &cobra.Command{
+		Use:   "ec2ssm",
+		Args:  cobra.NoArgs,
+		Short: "Use SSM to securely install on EC2 instances",
+		RunE:  installAWSSSM,
+		Long: `This command installs the agent on all EC2 instances in an AWS account using SSM.
+
+This command will create a role and instance profile with 'SSMManagedInstanceCore'
+attached and associate that instance profile with the target instances. If the target
+instances already have associated instance profiles, this command will not change
+their state. This command will teardown the IAM role and instance profile before exiting.
+
+This command authenticates with AWS credentials from well-known locations on the user's
+machine. The principal associated with these credentials should have the
+'AmazonEC2FullAccess', 'IAMFullAccess' and 'AmazonSSMFullAccess' policies attached.
+
+To skip IAM role / instance profile creation and instance profile association:
+
+    lacework agent aws-install ec2ssm --skip_infra_creation
```

Comment:
Given that the other flag is called `--iam_role_name`, perhaps call this:

```suggestion
    lacework agent aws-install ec2ssm --skip_iam_role_creation
```

I still think `infra` is too generic and does not inform the customer about what creation is being skipped.

Reply:
That makes sense, fixed this in 1659b5a60b61f3d313a22c7ce13c8777548521bc